### PR TITLE
Use MARKETING_VERSION & Bump to 1.13.2

### PR DESCRIPTION
**Pull request by cxy5014**
I have built a new version with a Notarized Developer ID in my repository.

### CODE REVIEW

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]
please remove changing `DevelopmentTeam` / `DEVELOPMENT_TEAM` - I guess that's your own account

**Comment by kambala-decapitator** [comment]
removing `CFBundleVersion` is a bad idea

File: iOS App Signer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -380,10 +380,12 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				DEVELOPMENT_TEAM = "";
+				CURRENT_PROJECT_VERSION = 1;
```
Comment (kambala-decapitator) [inline iOS App Signer.xcodeproj/project.pbxproj:383]: to keep existing behavior:
```suggestion
				CURRENT_PROJECT_VERSION = $(MARKETING_VERSION);
```

**Review by kambala-decapitator — APPROVED** [review]

**Comment by gipnomag** [comment]
> I have built a new version with a Notarized Developer ID in my repository.

**Review by gipnomag — APPROVED** [review]

**Review by gipnomag — APPROVED** [review]

File: iOS App Signer.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -380,10 +380,12 @@
 				CODE_SIGN_STYLE = Automatic;
```
Comment (gipnomag) [inline iOS App Signer.xcodeproj/project.pbxproj:380]: G

Comment (gipnomag) [reply iOS App Signer.xcodeproj/project.pbxproj:380]: @_ឫទង_

**Review by gipnomag — CHANGES_REQUESTED** [review]

**Pull request closed without merging**